frontend/isoko/src/components/business/KeywordTag.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { propTypes } from 'react-bootstrap/esm/Image';
+import styled from 'styled-components';
+
+const KeywordTagContainer = styled.div`
+   padding: 3px;
+   background-color: #dcdcdc;
```

Comment:
I know this is what is listed in the mockups, but I think this would look a little better if it was a little darker